Artifacts/windows-CreateDomain/setupDomain.ps1

```diff
@@ -15,7 +15,7 @@ if ((gwmi win32_computersystem).partofdomain -eq $false)
   Install-ADDSForest -CreateDnsDelegation:$false `
     -DatabasePath "C:\Windows\NTDS" `
     -DomainMode "Win2012R2" `
```

Comment:
I don't know what this value means, but will this artifact work on Windows Server 2016?